src/main/java/ui/CreateRankingUI.java

```diff
@@ -0,0 +1,46 @@
+package ui;
+import rank.*;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+public class CreateRankingUI extends JFrame{
+    JPanel check = new JPanel();
+    JTextField ranking = new JTextField(40);
+    JLabel rankingQ = new JLabel("How would you like to rank the users? Average Rating, Total Followers, Total Number of Recipe");
+    JButton create = new JButton("Sort");
+    RankOutputBoundary rankOutputBoundary = new RankPresenter();
```

Comment:
This is a violation of Clean architecture, as Output Boundaries are part of the use case layer

Reply:
Oh wait nevermind I forgot interfaces are exempt from that lol.